### PR TITLE
remove tests to isolate

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
Stiffness detection isn't switching back. Could be W-transform stuff?

Two event detection test misses.

### CODE REVIEW

**Comment by kanav99** [comment]
Setting `adaptive=false` gives correct results.
```
using OrdinaryDiffEq
f(du,u,p,t) = du .= u
prob = ODEProblem(f,[1.0], (0.0, 1.0))
sol = solve(prob, ImplicitEuler(); dt = 0.001, adaptive=false)
@show sol.u
```
Output-
```
sol.u = Array{Float64,1}[[1.0], [1.001], [1.002], [1.00301], [1.00401], [1.00502], [1.00602], [1.00703], [1.00804], [1.00905], [1.01006], [1.01107], [1.01208], [1.01309], [1.01411], [1.01512], [1.01614], [1.01715], [1.01817], [1.01919], [1.02021], [1.02123], [1.02226], [1.02328], [1.0243], [1.02533], [1.02635], [1.02738], [1.02841], [1.02944], [1.03047], [1.0315], [1.03253], [1.03357], [1.0346], [1.03564], [1.03667], [1.03771], [1.03875], [1.03979], [1.04083], [1.04187], [1.04292], [1.04396], [1.04501], [1.04605], [1.0471], [1.04815], [1.0492], [1.05025], [1.0513], [1.05235], [1.0534], [1.05446], [1.05551], [1.05657], [1.05763], [1.05869], [1.05975], [1.06081], [1.06187], [1.06293], [1.064], [1.06506], [1.06613], [1.06719], [1.06826], [1.06933], [1.0704], [1.07147], [1.07255], [1.07362], [1.07469], [1.07577], [1.07685], [1.07792], [1.079], [1.08008], [1.08116], [1.08225], [1.08333], [1.08441], [1.0855], [1.08659], [1.08767], [1.08876], [1.08985], [1.09094], [1.09204], [1.09313], [1.09422], [1.09532], [1.09642], [1.09751], [1.09861], [1.09971], [1.10081], [1.10191], [1.10302], [1.10412], [1.10523], [1.10633], [1.10744], [1.10855], [1.10966], [1.11077], [1.11188], [1.11299], [1.11411], [1.11522], [1.11634], [1.11746], [1.11858], [1.1197], [1.12082], [1.12194], [1.12306], [1.12419], [1.12531], [1.12644], [1.12756], [1.12869], [1.12982], [1.13095], [1.13209], [1.13322], [1.13435], [1.13549], [1.13663], [1.13776], [1.1389], [1.14004], [1.14118], [1.14233], [1.14347], [1.14461], [1.14576], [1.14691], [1.14805], [1.1492], [1.15035], [1.15151], [1.15266], [1.15381], [1.15497], [1.15612], [1.15728], [1.15844], [1.1596], [1.16076], [1.16192], [1.16308], [1.16425], [1.16541], [1.16658], [1.16775], [1.16892], [1.17009], [1.17126], [1.17243], [1.1736], [1.17478], [1.17596], [1.17713], [1.17831], [1.17949], [1.18067], [1.18185], [1.18304], [1.18422], [1.18541], [1.18659], [1.18778], [1.18897], [1.19016], [1.19135], [1.19254], [1.19374], [1.19493], [1.19613], [1.19733], [1.19852], [1.19972], [1.20092], [1.20213], [1.20333], [1.20453], [1.20574], [1.20695], [1.20816], [1.20936], [1.21058], [1.21179], [1.213], [1.21421], [1.21543], [1.21665], [1.21786], [1.21908], [1.2203], [1.22152], [1.22275], [1.22397], [1.2252], [1.22642], [1.22765], [1.22888], [1.23011], [1.23134], [1.23257], [1.23381], [1.23504], [1.23628], [1.23752], [1.23876], [1.24], [1.24124], [1.24248], [1.24372], [1.24497], [1.24621], [1.24746], [1.24871], [1.24996], [1.25121], [1.25246], [1.25372], [1.25497], [1.25623], [1.25749], [1.25874], [1.26], [1.26127], [1.26253], [1.26379], [1.26506], [1.26632], [1.26759], [1.26886], [1.27013], [1.2714], [1.27267], [1.27395], [1.27522], [1.2765], [1.27778], [1.27906], [1.28034], [1.28162], [1.2829], [1.28419], [1.28547], [1.28676], [1.28805], [1.28934], [1.29063], [1.29192], [1.29321], [1.29451], [1.2958], [1.2971], [1.2984], [1.2997], [1.301], [1.3023], [1.3036], [1.30491], [1.30621], [1.30752], [1.30883], [1.31014], [1.31145], [1.31277], [1.31408], [1.3154], [1.31671], [1.31803], [1.31935], [1.32067], [1.32199], [1.32332], [1.32464], [1.32597], [1.32729], [1.32862], [1.32995], [1.33128], [1.33262], [1.33395], [1.33528], [1.33662], [1.33796], [1.3393], [1.34064], [1.34198], [1.34332], [1.34467], [1.34602], [1.34736], [1.34871], [1.35006], [1.35141], [1.35277], [1.35412], [1.35548], [1.35683], [1.35819], [1.35955], [1.36091], [1.36227], [1.36364], [1.365], [1.36637], [1.36774], [1.3691], [1.37048], [1.37185], [1.37322], [1.37459], [1.37597], [1.37735], [1.37873], [1.38011], [1.38149], [1.38287], [1.38426], [1.38564], [1.38703], [1.38842], [1.38981], [1.3912], [1.39259], [1.39398], [1.39538], [1.39678], [1.39817], [1.39957], [1.40098], [1.40238], [1.40378], [1.40519], [1.40659], [1.408], [1.40941], [1.41082], [1.41223], [1.41365], [1.41506], [1.41648], [1.4179], [1.41932], [1.42074], [1.42216], [1.42358], [1.42501], [1.42643], [1.42786], [1.42929], [1.43072], [1.43215], [1.43359], [1.43502], [1.43646], [1.4379], [1.43934], [1.44078], [1.44222], [1.44366], [1.44511], [1.44655], [1.448], [1.44945], [1.4509], [1.45236], [1.45381], [1.45526], [1.45672], [1.45818], [1.45964], [1.4611], [1.46256], [1.46403], [1.46549], [1.46696], [1.46843], [1.4699], [1.47137], [1.47284], [1.47432], [1.47579], [1.47727], [1.47875], [1.48023], [1.48171], [1.48319], [1.48468], [1.48616], [1.48765], [1.48914], [1.49063], [1.49212], [1.49362], [1.49511], [1.49661], [1.49811], [1.49961], [1.50111], [1.50261], [1.50411], [1.50562], [1.50713], [1.50864], [1.51015], [1.51166], [1.51317], [1.51469], [1.5162], [1.51772], [1.51924], [1.52076], [1.52228], [1.52381], [1.52533], [1.52686], [1.52839], [1.52992], [1.53145], [1.53298], [1.53451], [1.53605], [1.53759], [1.53913], [1.54067], [1.54221], [1.54375], [1.5453], [1.54685], [1.54839], [1.54994], [1.5515], [1.55305], [1.5546], [1.55616], [1.55772], [1.55928], [1.56084], [1.5624], [1.56396], [1.56553], [1.5671], [1.56867], [1.57024], [1.57181], [1.57338], [1.57496], [1.57653], [1.57811], [1.57969], [1.58127], [1.58285], [1.58444], [1.58602], [1.58761], [1.5892], [1.59079], [1.59238], [1.59398], [1.59557], [1.59717], [1.59877], [1.60037], [1.60197], [1.60358], [1.60518], [1.60679], [1.6084], [1.61001], [1.61162], [1.61323], [1.61485], [1.61646], [1.61808], [1.6197], [1.62132], [1.62294], [1.62457], [1.6262], [1.62782], [1.62945], [1.63108], [1.63272], [1.63435], [1.63599], [1.63762], [1.63926], [1.6409], [1.64255], [1.64419], [1.64584], [1.64748], [1.64913], [1.65078], [1.65244], [1.65409], [1.65575], [1.6574], [1.65906], [1.66072], [1.66239], [1.66405], [1.66572], [1.66738], [1.66905], [1.67072], [1.6724], [1.67407], [1.67575], [1.67742], [1.6791], [1.68078], [1.68247], [1.68415], [1.68584], [1.68752], [1.68921], [1.6909], [1.6926], [1.69429], [1.69599], [1.69768], [1.69938], [1.70108], [1.70279], [1.70449], [1.7062], [1.70791], [1.70961], [1.71133], [1.71304], [1.71475], [1.71647], [1.71819], [1.71991], [1.72163], [1.72335], [1.72508], [1.72681], [1.72853], [1.73026], [1.732], [1.73373], [1.73547], [1.7372], [1.73894], [1.74068], [1.74242], [1.74417], [1.74591], [1.74766], [1.74941], [1.75116], [1.75292], [1.75467], [1.75643], [1.75819], [1.75995], [1.76171], [1.76347], [1.76524], [1.767], [1.76877], [1.77054], [1.77231], [1.77409], [1.77586], [1.77764], [1.77942], [1.7812], [1.78299], [1.78477], [1.78656], [1.78835], [1.79014], [1.79193], [1.79372], [1.79552], [1.79731], [1.79911], [1.80091], [1.80272], [1.80452], [1.80633], [1.80814], [1.80995], [1.81176], [1.81357], [1.81539], [1.8172], [1.81902], [1.82084], [1.82267], [1.82449], [1.82632], [1.82814], [1.82997], [1.83181], [1.83364], [1.83548], [1.83731], [1.83915], [1.84099], [1.84284], [1.84468], [1.84653], [1.84838], [1.85023], [1.85208], [1.85393], [1.85579], [1.85765], [1.8595], [1.86137], [1.86323], [1.86509], [1.86696], [1.86883], [1.8707], [1.87257], [1.87445], [1.87632], [1.8782], [1.88008], [1.88196], [1.88385], [1.88573], [1.88762], [1.88951], [1.8914], [1.8933], [1.89519], [1.89709], [1.89899], [1.90089], [1.90279], [1.9047], [1.9066], [1.90851], [1.91042], [1.91233], [1.91425], [1.91616], [1.91808], [1.92], [1.92192], [1.92385], [1.92577], [1.9277], [1.92963], [1.93156], [1.9335], [1.93543], [1.93737], [1.93931], [1.94125], [1.94319], [1.94514], [1.94708], [1.94903], [1.95098], [1.95294], [1.95489], [1.95685], [1.95881], [1.96077], [1.96273], [1.9647], [1.96666], [1.96863], [1.9706], [1.97257], [1.97455], [1.97653], [1.9785], [1.98048], [1.98247], [1.98445], [1.98644], [1.98843], [1.99042], [1.99241], [1.9944], [1.9964], [1.9984], [2.0004], [2.0024], [2.00441], [2.00641], [2.00842], [2.01043], [2.01244], [2.01446], [2.01647], [2.01849], [2.02051], [2.02254], [2.02456], [2.02659], [2.02862], [2.03065], [2.03268], [2.03471], [2.03675], [2.03879], [2.04083], [2.04287], [2.04492], [2.04697], [2.04901], [2.05107], [2.05312], [2.05517], [2.05723], [2.05929], [2.06135], [2.06341], [2.06548], [2.06755], [2.06962], [2.07169], [2.07376], [2.07584], [2.07792], [2.08], [2.08208], [2.08416], [2.08625], [2.08834], [2.09043], [2.09252], [2.09461], [2.09671], [2.09881], [2.10091], [2.10301], [2.10512], [2.10723], [2.10934], [2.11145], [2.11356], [2.11568], [2.11779], [2.11991], [2.12204], [2.12416], [2.12629], [2.12842], [2.13055], [2.13268], [2.13481], [2.13695], [2.13909], [2.14123], [2.14337], [2.14552], [2.14767], [2.14982], [2.15197], [2.15412], [2.15628], [2.15844], [2.1606], [2.16276], [2.16493], [2.16709], [2.16926], [2.17143], [2.17361], [2.17578], [2.17796], [2.18014], [2.18232], [2.18451], [2.18669], [2.18888], [2.19107], [2.19327], [2.19546], [2.19766], [2.19986], [2.20206], [2.20427], [2.20647], [2.20868], [2.21089], [2.21311], [2.21532], [2.21754], [2.21976], [2.22198], [2.22421], [2.22643], [2.22866], [2.23089], [2.23312], [2.23536], [2.2376], [2.23984], [2.24208], [2.24432], [2.24657], [2.24882], [2.25107], [2.25332], [2.25558], [2.25784], [2.2601], [2.26236], [2.26462], [2.26689], [2.26916], [2.27143], [2.27371], [2.27598], [2.27826], [2.28054], [2.28282], [2.28511], [2.2874], [2.28969], [2.29198], [2.29427], [2.29657], [2.29887], [2.30117], [2.30347], [2.30578], [2.30809], [2.3104], [2.31271], [2.31502], [2.31734], [2.31966], [2.32198], [2.32431], [2.32663], [2.32896], [2.33129], [2.33363], [2.33596], [2.3383], [2.34064], [2.34299], [2.34533], [2.34768], [2.35003], [2.35238], [2.35474], [2.35709], [2.35945], [2.36181], [2.36418], [2.36654], [2.36891], [2.37128], [2.37366], [2.37603], [2.37841], [2.38079], [2.38318], [2.38556], [2.38795], [2.39034], [2.39273], [2.39513], [2.39753], [2.39993], [2.40233], [2.40473], [2.40714], [2.40955], [2.41196], [2.41438], [2.41679], [2.41921], [2.42163], [2.42406], [2.42648], [2.42891], [2.43134], [2.43378], [2.43621], [2.43865], [2.44109], [2.44354], [2.44598], [2.44843], [2.45088], [2.45334], [2.45579], [2.45825], [2.46071], [2.46317], [2.46564], [2.46811], [2.47058], [2.47305], [2.47553], [2.47801], [2.48049], [2.48297], [2.48545], [2.48794], [2.49043], [2.49293], [2.49542], [2.49792], [2.50042], [2.50292], [2.50543], [2.50794], [2.51045], [2.51296], [2.51547], [2.51799], [2.52051], [2.52304], [2.52556], [2.52809], [2.53062], [2.53315], [2.53569], [2.53823], [2.54077], [2.54331], [2.54586], [2.54841], [2.55096], [2.55351], [2.55607], [2.55862], [2.56119], [2.56375], [2.56632], [2.56888], [2.57146], [2.57403], [2.57661], [2.57919], [2.58177], [2.58435], [2.58694], [2.58953], [2.59212], [2.59472], [2.59731], [2.59991], [2.60252], [2.60512], [2.60773], [2.61034], [2.61295], [2.61557], [2.61818], [2.62081], [2.62343], [2.62606], [2.62868], [2.63132], [2.63395], [2.63659], [2.63923], [2.64187], [2.64451], [2.64716], [2.64981], [2.65246], [2.65512], [2.65777], [2.66043], [2.6631], [2.66576], [2.66843], [2.6711], [2.67378], [2.67645], [2.67913], [2.68181], [2.6845], [2.68719], [2.68988], [2.69257], [2.69526], [2.69796], [2.70066], [2.70337], [2.70607], [2.70878], [2.71149], [2.71421], [2.71692], [2.71964]]
```

**Comment by ChrisRackauckas** [comment]
Which test is that?

**Comment by kanav99** [comment]
No, not any test, just the basic script posted above gives wrong results for `adaptive=true`.

**Comment by kanav99** [comment]
Its somewhere sommething about mismatch between `W_dt` and `dt` IMO. Uncommenting the relaxation line in NLNewton gives _better_ results.

**Comment by ChrisRackauckas** [comment]
It also solves with adaptive?

**Comment by kanav99** [comment]
It does solve but I get this -
```
using OrdinaryDiffEq
f(du,u,p,t) = du .= u
prob = ODEProblem(f,[1.0], (0.0, 1.0))
sol = solve(prob, ImplicitEuler(); dt = 0.001)
@show sol.u
```
Out-
```
sol.u = Array{Float64,1}[[1.0], [1.001], [1.002], [1.01212], [1.01221], [1.01306], [1.02164], [1.11532], [1.11536], [1.1154], [1.11578], [1.11958], [1.15886], [1.21999], [1.28436], [1.30399], [1.53512], [1.53524], [1.53537], [1.53564], [1.5364], [1.5441], [1.54422], [1.54549], [1.55824], [1.69714], [1.6972], [1.69727], [1.69787], [1.70392], [1.76661], [1.85539], [1.8572], [1.87547], [2.07781], [2.07786], [2.07792], [2.07846], [2.08392], [2.13997], [2.14069], [2.14787], [2.22212], [2.34107], [2.46638], [2.46869], [2.49201], [2.74926], [2.74934], [2.74942], [2.75019], [2.75791], [2.83726], [2.83834], [2.84926], [2.9628], [3.08086], [3.0836], [3.08569], [3.08785], [3.10962], [3.10986], [3.11234], [3.13728], [3.40828], [3.40842], [3.40857], [3.40984], [3.42262], [3.5554], [3.72471], [3.72794], [3.7605], [4.11699], [4.11714], [4.11728], [4.1186], [4.13185], [4.26869], [4.27098], [4.29396], [4.53671], [4.53708], [4.53745], [4.53982], [4.55385], [4.69853], [4.70074], [4.72291], [4.95556], [4.95597], [4.95638], [4.95905], [4.98296], [4.98337], [4.98747], [5.02879], [5.47899], [5.4792], [5.47942], [5.48135], [5.50071], [5.70145], [5.99025], [5.99692], [6.06436], [6.06484], [6.0696], [6.11763], [6.63877], [6.63906], [6.63935], [6.64189], [6.66735], [6.93215], [6.93823], [6.99959], [7.65845], [7.65873], [7.65901], [7.66153], [7.68689], [7.94916], [8.36037]]

```
Which is nowhere close to correct answer.

**Comment by ChrisRackauckas** [comment]
Its error estimator isn't great. Just lower the tolerance.

**Comment by kanav99** [comment]
Oh yes. Didn't knew about this. Sorry I just ended up trying to find a false bug :stuck_out_tongue:

**Comment by ChrisRackauckas** [comment]
Looks like events were fine @kanav99

File: src/dense/stiff_addsteps.jl [hunk 1]
```diff
@@ -27,33 +28,28 @@ function DiffEqBase.addsteps!(k,t,uprev,u,dt,f,p,cache::Union{Rosenbrock23Cache,
...
-      @inbounds linsolve_tmp[i] = @muladd fsalfirst[i] + γ*dT[i]
-    end
+    @.. linsolve_tmp = @muladd fsalfirst + γ*dT
 
     ### Jacobian does not need to be re-evaluated after an event
     ### Since it's unchanged
-    for i in 1:length(u), j in 1:length(u)
-      @inbounds W[i,j] = @muladd I[i,j]-γ*J[i,j]
-    end
+    jacobian2W!(W, mass_matrix, γ, J, false)
```
Comment (ChrisRackauckas) [inline src/dense/stiff_addsteps.jl]: @YingboMa this might change the mass matrix sensitivity results since now MM is involved in the `addsteps!`

**Pull request merged**